app/views/facility_orders/index.html.haml

```diff
@@ -2,46 +2,46 @@
   = current_facility
 
 = content_for :tabnav do
-  = render :partial => 'admin/shared/tabnav_order', :locals => { :secondary_tab => 'new'}
+  = render partial: "admin/shared/tabnav_order", locals: { secondary_tab: "new"}
 
 - if @order_details.empty?
   %p.notice There are no "New" or "In Process" orders.
 - else
-  = form_tag(batch_update_facility_orders_path, :method => :post) do
+  = form_tag(batch_update_facility_orders_path, method: :post) do
     - facility_users = User.find_users_by_facility(current_facility)
     %table.table.table-striped.table-hover.old-table
-      = render partial: 'table_controls'
+      = render partial: "table_controls"
       %thead
         -## THE HEADER ROW
         %tr
           %th
-          %th.order_num_width.centered= sortable 'order_number', 'Order'
-          %th.order_num_width.centered= 'Order Detail'
-          %th.ordered_on_width= sortable 'date', 'Ordered On'
+          %th.order_num_width.centered= sortable "order_number", "Order"
+          %th.order_num_width.centered= "Order Detail"
```

Comment:
Should we do any I18n while we're here?

Reply:
I'm leaning toward doing i18n in a separate PR. There's just a ton of i18n in some of these older views.

Reply:
I'm cool with doing I18n separately.